pkg/config/prow-config-documented.yaml

```diff
@@ -1355,6 +1355,13 @@ scheduler:
         # configured to in the first place.
         mappings:
             "": ""
+    rest_by_job:
+        # Interval is the interval to clean up the cache
+        interval: 0s
+        # Timeout is the timeout for individual cache entry
+        timeout: 0s
```

Comment:
And for this as well. Since both are configurations for the cache only, perhaps you can consider adding them under a `cache` stanza.

Reply:
good idea